crypto/bech32.go

```diff
@@ -0,0 +1,122 @@
+package crypto
```

Comment:
Could you add some comments that we copied code from CodeChain JS SDK?

Reply:
I added it. Is this enough?